engine/gamesys/src/gamesys/components/comp_gui.cpp

```diff
@@ -1123,20 +1123,66 @@ namespace dmGameSystem
                 continue;
             }
 
+            // pre-multiplied alpha
             const Vector4& color = dmGui::GetNodeProperty(scene, node, dmGui::PROPERTY_COLOR);
-
-            // Pre-multiplied alpha
             Vector4 pm_color(color.getXYZ(), node_opacities[i]);
-            Vector4 slice9 = dmGui::GetNodeSlice9(scene, node);
-            Point3 size = dmGui::GetNodeSize(scene, node);
 
+            // default not uv_rotated texture coords
+            const float default_tc[6] = {0, 0, 0, 1, 1, 1};
+            const float* tc = dmGui::GetNodeFlipbookAnimUV(scene, node);
+
+            bool manually_set_texture = tc == 0;
```

Comment:
I think this variable needs commenting.
When is tc == 0 ?

Reply:
Fixed in: https://github.com/defold/defold/pull/4921/commits/ee8b54705e82bd3f14a3c75619f23bdeaf0f2fa0